srcpkgs/chez-exe/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'chez-exe'
+pkgname=chez-exe
+version=0.1.1
+revision=1
+_chezversion=9.5.2
+archs="x86_64 i686"
```

Comment:
Any reason this wouldn't work on musl?

Reply:
Now it should work with the musl targets: forgot to add them. I've tried to match the supported targets by Chez Scheme.